packages/shared/src/enterprise/licenseUtil.ts

```diff
@@ -147,6 +158,19 @@ export interface LicenseInterface {
     discountMessage?: string; // The message of the discount
     hasPaymentMethod?: boolean;
   };
+  orbSubscription?: {
+    id: string;
+    customerId: string;
+    qty: number;
+    trialEnd: Date | null;
+    status: Stripe.Subscription.Status;
```

Comment:
Orb has slightly different statuses (`Orb.Subscription.Status`), which can be "active", "ended", or "upcoming".  Whereas Stripe has `active`, `canceled`, `incomplete`, `incomplete_expired`, `past_due`, `paused`, `trialing`, `unpaid`.

I can map Orb's statuses to Stripes if we want to align on Stripe's status types.
"active" => "active"
"ended" => "cancelled"
"upcoming" => "paused" - we don't have a concept of this today, and don't anticipate we will in the future.

Reply:
yeah, let's align on Stripe's statuses